src/pages/TeamCalendar/TeamCalendar.tsx

```diff
@@ -0,0 +1,55 @@
+import { useState } from 'react';
+import Drawer from '@/components/common/Drawer/Drawer';
+import TeamMembers from '@/pages/TeamCalendar/sections/TeamMembers';
+import RecommendTimes from '@/pages/TeamCalendar/sections/RecommendTimes';
+import FullCalendar from '@/pages/Calendar/FullCalendar';
+import TeamListModal from '@/pages/TeamCalendar/sections/TeamListModal';
+
+const TeamCalendarPage = () => {
+  const [isModalOpen, setIsModalOpen] = useState(false);
```

Comment:
```suggestion
  const [isTeamListModalOpen, setIsTeamListModalOpen] = useState(false);
```

TeamListModal 컴포넌트를 제어하기 위한 상태이므로, 변수명도 명확히 지정해주면 좋습니다.
나중에 모달이 더 많아질 수도 있구요~!